### ISSUE TITLE
Looks like @types/history is not valid

### ISSUE THREAD

**Issue by vslinko** [Microsoft/types-publisher#91]
I've got errors after [`@types/history`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/types-2.0/history/index.d.ts) is installed:

```
node_modules/@types/history/index.d.ts(120,42): error TS2307: Cannot find module 'history/lib/createBrowserHistory'.
node_modules/@types/history/index.d.ts(121,46): error TS2307: Cannot find module 'history/lib/createHashHistory'.
node_modules/@types/history/index.d.ts(122,48): error TS2307: Cannot find module 'history/lib/createMemoryHistory'.
node_modules/@types/history/index.d.ts(123,43): error TS2307: Cannot find module 'history/lib/createLocation'.
node_modules/@types/history/index.d.ts(124,40): error TS2307: Cannot find module 'history/lib/useBasename'.
node_modules/@types/history/index.d.ts(125,44): error TS2307: Cannot find module 'history/lib/useBeforeUnload'.
node_modules/@types/history/index.d.ts(126,39): error TS2307: Cannot find module 'history/lib/useQueries'.
node_modules/@types/history/index.d.ts(127,26): error TS2307: Cannot find module 'history/lib/actions'.
```

Looks like `lib` folder from DefinitelyTyped repository is not included into `@types/history` module.

**Comment by vslinko** [comment]
FYI: I've created PR with `history` module definition updates https://github.com/DefinitelyTyped/DefinitelyTyped/pull/10261

**Comment by brunolm** [comment]
Same issue here

![bug](https://i.imgur.com/2edSirq.png)

**Comment by mhegazy** [comment]
should be fixed in `@types/history@2.0.31`
